modules/virt-about-ssp-operator.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * virt/virt-architecture.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-ssp-operator_{context}"]
+= About ssp-operator
+
+The `ssp-operator` deploys the common templates, the related default boot sources, and the template validator.
```

Comment:
The figure is not in sync with the text.

Reply:
In in images/cnv_components_ssp-operator.png  only the template validator is shown, but the common templates and  the related default boot sources are missing.

Reply:
@sabrinajess 
If you want to update any of the diagrams, let me know and I will create separate tasks for the Tech Debt post-4.11 epic. Thanks.